movie-react-app/src/components/SearchPage.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from "react";
+
+class SearchPage extends Component {
+  constructor(props) {
+    super(props);
+    this.onSearch = this.onSearch.bind(this);
+    this.renderMovie = this.renderMovie.bind(this);
+    this.state = {
+      searchText: "",
+      results: []
+    };
+  }
+
+  onSearch() {
+    fetch(
+      `http://www.omdbapi.com/?apikey=aecbec56&s=${encodeURIComponent(
+        this.state.searchText
+      )}`
+    )
+      .then(response => response.json())
+      .then(body => {
+        const results = body.Search;
+        // check body.Error? i.e. "Too many results"
```

Comment:
If you type something like `ab` and search, it'll return an error in the body. Probably we want to handle it but just keeping it simple for now